README.md

```diff
@@ -79,10 +81,21 @@ Option | Default value | Description
 ------ | ------------- | -----------
 url | wss://www.streamr.com/api/v1/ws | Address of the Streamr websocket endpoint to connect to.
 restUrl | https://www.streamr.com/api/v1/ws | Base URL of the Streamr REST API.
-apiKey | null | Define a default apiKey to use when none is specified in the individual function calls.
+apiKey | null | Defined for backwards compatibility. Should use `auth.apiKey` instead.
```

Comment:
I think we can leave this line out of the readme. Old functionality will still work, but let's not point people to it anymore.